inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/EvaluationSimulationPanel.java

```diff
@@ -0,0 +1,218 @@
+
+        LearningCurve learningCurve = new LearningCurve();
+        learningCurve.setCurveData(curveData);
+        learningCurve.setXaxis(trainingSizes);
+
+        return learningCurve;
+    }
+    
+    /**
+     * evaluates the selected recommender with the help of the annotated documents in the project. 
+     * 
+     * @param aTarget uses to log errors on the page in case of unwanted behaviour
+     * 
+     * @return comma separated string of scores in the first index of the array
+     */
+    private String[] evaluate()
+    {
+        //there must be some recommender selected by the user on the UI
+        if (selectedRecommenderPanel.getObject() == null
+                || selectedRecommenderPanel.getObject().getTool() == null) {
+            LOG.error("Please select a recommender from the list");
+            return null;
+        }
+
+        //get all the source documents related to the project
+        Map<SourceDocument, AnnotationDocument> listAllDocuments = documentService
+                .listAllDocuments(project, userDao.getCurrentUser());
+
+        //create a list of CAS from the pre-annotated documents of the project
+        List<CAS> casList = new ArrayList<>();
+        listAllDocuments.forEach((source, annotation) -> {
+            try {
+                CAS cas = documentService.createOrReadInitialCas(source);
+                casList.add(cas);
+            }
+            catch (IOException e1) {
+                LOG.error("Unable to render chart", e1);
+                return;
+            }
+        });
+        
+        IncrementalSplitter splitStrategy = new IncrementalSplitter(TRAIN_PERCENTAGE,
+                INCREMENT, LOW_SAMPLE_THRESHOLD);
+
+        @SuppressWarnings("rawtypes")
+        RecommendationEngineFactory factory = recommenderRegistry
+                .getFactory(selectedRecommenderPanel.getObject().getTool());
+        RecommendationEngine recommender = factory.build(selectedRecommenderPanel.getObject());
+        
+        if (recommender == null)
+        {
+            LOG.warn("Unknown Recommender selected");
+            return null;
+        }
+        
+        StringBuilder sbScore = new StringBuilder();
+        StringBuilder sbTrainingSize = new StringBuilder();
+
+        // create a list of comma separated string of scores from every iteration of
+        // evaluation.
+        while (splitStrategy.hasNext()) {
+            splitStrategy.next();
+
+            double trainingSize;
+            double score;
+            try {
+                EvaluationResult evaluationResult = recommender.evaluate(casList, splitStrategy);
+                
+                if (evaluationResult.isEvaluationSkipped())                {
+                    LOG.warn("Evaluation skipped. Chart cannot to be shown");
+                    continue;
+                }
+                
+                score = evaluationResult.getDefaultScore();
```

Comment:
I deleted `evaluationResult.getDefaultScore()`, you can use `evaluationResult.computeF1Score()`.